classes/class-schedule.php

```diff
@@ -690,17 +688,7 @@ public function get_interval() {
 	 * @return array
 	 */
 	public static function get_cron_schedules() {
-
-		$schedules = wp_get_schedules();
-
-		// remove any schedule whose key is not prefixed with 'hmbkp_'
-		foreach ( $schedules as $key => $arr ) {
-			if ( ! preg_match( '/^hmbkp_/', $key ) ) {
-				unset( $schedules[ $key ] );
-			}
-		}
-
-		return $schedules;
+		return hmbkp_cron_schedules();
```

Comment:
Might it make sense to just move the contents of the `hmbkp_cron_schedules` function into `::get_cron_schedules` and do away with `hmbkp_cron_schedules` altogether? It's `static` so can easily be used externally.